google/cloud/forseti/services/base/config.py

```diff
@@ -148,15 +149,50 @@ def get_storage_class(self):
 
 
 class InventoryConfig(AbstractInventoryConfig):
+    # pylint: disable=too-many-instance-attributes
     """Implements composed dependency injection for the inventory."""
 
+    @staticmethod
+    def _filter_valid_resources(excluded_resources):
```

Comment:
I recommend adding a few tests for InventoryConfig that exercizes this function to ensure that it is properly filtering out bad resources, and handling the folders/ -> folder/ conversion correctly, etc.

Reply:
Added unit tests to this method